LoopKit/InsulinKit/CachedInsulinDeliveryObject+CoreDataClass.swift

```diff
@@ -82,6 +82,35 @@ class CachedInsulinDeliveryObject: NSManagedObject {
             primitiveProgrammedTempBasalRate = NSNumber(value: rate)
         }
     }
+
+    var insulinModelCategory: InsulinModelCategory? {
```

Comment:
We aren't storing a category of insulin model in HK, but rather a field that represents the *type* of insulin. The insulin *model* that Loop uses may be related to the type of insulin, and loop may switch models based on the type of insulin. But I don't think we should call this a `insulinModelCategory`.  Something like `insulinType` would be better here.

Reply:
Thanks so much for the suggestion - I was struggling to think of a better name for it!